atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -230,11 +232,11 @@ public void increaseInShardsReflectedOnLastSweptTimestamp() {
     public void callingEnqueueAndSweepOnUninitializedSweeperThrows() {
         TargetedSweeper uninitializedSweeper = TargetedSweeper.createUninitializedForTest(() -> 1);
         assertThatThrownBy(() -> uninitializedSweeper.enqueue(ImmutableList.of()))
+                .isInstanceOf(NotInitializedException.class);
```

Comment:
consider: also making this one an assertThatLoggableExceptionThrownBy that verifies it also returns the same args (e.g., just pulling the assert into a separate method and doing smth like assertThatNotInitializedExceptionThrownBy